pkgs/development/python-modules/superqt/default.nix

```diff
@@ -1,44 +1,44 @@
 { lib
 , buildPythonPackage
 , fetchFromGitHub
-, setuptools-scm
+, hatch-vcs
+, hatchling
+, pygments
 , pyqt5
 , qtpy
 , typing-extensions
-, pytestCheckHook
-, pygments
 }:
 
 buildPythonPackage rec {
   pname = "superqt";
-  version = "0.3.8";
-  format = "pyproject";
+  version = "0.6.1";
+  pyproject = true;
 
   src = fetchFromGitHub {
-    owner = "napari";
-    repo = pname;
-    rev = "refs/tags/v${version}";
-    hash = "sha256-zEMG2zscGDlRxtLn/lUTEjZBPabcwzMcj/kMcy3yOs8=";
+    owner = "pyapp-kit";
+    repo = "superqt";
+    rev = "v${version}";
+    hash = "sha256-kSu3QAr9GHmTkH9fyOuuI9UGSUgsYDUROOeQppXFQZg=";
   };
 
-  nativeBuildInputs = [ setuptools-scm ];
+  nativeBuildInputs = [
+    hatch-vcs
+    hatchling
+  ];
 
   propagatedBuildInputs = [
     pyqt5
+    pygments
     qtpy
     typing-extensions
-    pygments
   ];
 
-  nativeCheckInputs = [ pytestCheckHook ];
-
-  doCheck = false; # Segfaults...
-
```

Comment:
I want to keep these lines because we have no way of knowing if the test failed or if we simply forgot to run the test.